startGame.js

```diff
@@ -1,31 +1,16 @@
-
-
-
-function sectionStateOnStart (aboveSection,startButton,playScreen,gameBox){
+// This opens the gamescreen on click of the start button.
 $('#startButton').click(function () {
-    hideAboveSection (aboveSection)
-    hideStartButton (startButton)
-    showPlayScreen (playScreen)
-    increaseGameHeight (gameBox)
+    displayToggleNone ($('#aboveSection'))
+    displayToggleNone ($('#startButton'))
+    showPlayScreen ($('#playScreen'))
+    displayToggleNone($('.topSection'))
 })
 
+//Toggle display from block to none
+function displayToggleNone (section) {
+    section.css({display: "none"})
 }
-
-function hideAboveSection (aboveSection) {
-    aboveSection.css({display: "none"})
-}
-
-function hideStartButton (startButton) {
-    startButton.css({display: "none"})
-}
-
+//toggles the visibility of a screen from hidden to visible and brings to the front.
```

Comment:
i think this is the wrong use of toggle in your description - toggle looks into what the current state of your screen is and then does the opposite, e.g. display or hide, we are just allowing our code to change to visible